mlrun/api/api/endpoints/healthz.py

```diff
@@ -64,8 +65,11 @@ def _resolve_nuclio_version():
         # config override everything
         nuclio_version = config.nuclio_version
         if not nuclio_version and config.nuclio_dashboard_url:
-            nuclio_client = nuclio.Client()
-            nuclio_version = nuclio_client.get_dashboard_version()
+            try:
+                nuclio_client = nuclio.Client()
+                nuclio_version = nuclio_client.get_dashboard_version()
+            except Exception:
+                logger.warning("Failed to resolve nuclio version")
```

Comment:
We don't want to lose the exception data (why it failed)
```suggestion
            except Exceptionas exc:
                logger.warning("Failed to resolve nuclio version", exc=str(exc))
```